src/sentry/event_manager.py

```diff
@@ -2160,9 +2162,20 @@ def _save_aggregate_performance(jobs: Sequence[PerformanceJob], projects):
             ).select_related("group")
 
             new_grouphashes = set(group_hashes) - {hash.hash for hash in existing_grouphashes}
-            new_grouphashes_count = len(new_grouphashes)
 
             if new_grouphashes:
+                # temporary fix to limit group creation to grouphashes seen 3+ times in a 24-48 hour period
+                if settings.SENTRY_PERFORMANCE_ISSUES_REDUCE_NOISE:
+                    groups_to_create = new_grouphashes.copy()
+                    client = redis.redis_clusters.get("default")
```

Comment:
Let's make sure this is configurable with a fallback to `default`. e.g. https://github.com/getsentry/sentry/blob/master/src/sentry/reprocessing2.py#L465

🚨  In production the `default` is not a Redis cluster compatible so you either need a new one or pick an existing one and configure getsentry in advance.